dropwizard-jersey/src/main/java/io/dropwizard/jersey/validation/JerseyViolationExceptionMapper.java

```diff
@@ -12,8 +14,14 @@
 
 @Provider
 public class JerseyViolationExceptionMapper implements ExceptionMapper<JerseyViolationException> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(JerseyViolationExceptionMapper.class);
+
     @Override
     public Response toResponse(final JerseyViolationException exception) {
+        if (LOGGER.isDebugEnabled()) {
```

Comment:
The difference between

```java
LOG.debug("Something to say", throwable);

// and

if (LOG.isDebugEnabled()) {
    LOG.debug("Something to say", throwable);
}
```
should be so minimal in this case (we're talking about in the middle of a jersey exception handler!) that I would prefer more the concise approach. 

Thank you for making the changes.